.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Operating System:
+- Node Version:
+- NPM Version:
+- Rollup Version:
```

Comment:
Maybe put the Rollup version to the top as it is the most important piece here. As Rollup could theoretically be run in a browser, we might prefix the rest with an `if applicable:`.

Reply:
Done